generators/app/templates/_package.json

```diff
@@ -212,8 +212,9 @@
     "stylelint-config-standard": "~22.0.0",
     "stylelint-scss": "~3.20.1",
     "ts-node": "^10.1.0",
-    "typescript": "~4.3.2"
+    "typescript": "~4.3.2",
 <% if (props.tools.includes('prettier')) { -%>
+    "ts-jest": "^27.0.5"
```

Comment:
You should move this it before the typescript one, to avoid breaking the package.json generation when jest is not used.

Also, the conditional is incorrect here as it should be `props.tools.includes('jest')` and the previous prettier conditional should be left untouched.